editions/tw5.com/tiddlers/filters/search.tid

```diff
@@ -16,15 +16,15 @@ type: text/vnd.tiddlywiki
 The <<.op search>> operator uses an extended syntax that permits multiple fields and flags to be passed:
 
 ```
-[search:<field list>:<flag list>[<operand>]]
+[search:<field list>:<flag list>[<parameter>]]
 ```
 
 * ''field list'': a comma delimited list of field names to restrict the search
 ** defaults to <<.field tags>>, <<.field text>> and <<.field title>> if blank
 ** an asterisk `*` instead of the field list causes the search to be performed across all fields available on each tiddler
 ** preceding the list with a minus sign `-` reverses the order so that the search is performed on all fields except the listed fields
 * ''flag list'': a comma delimited list of flags (defaults to `words` if blank)
-* ''operand'': filter operand
+
```

Comment:
Is this additional blank line intentional? Shouldn't we be describing "parameter" instead?